pkg/operator/apiserver/audit/testdata/audit-policies-cm-scenario-1.yaml

```diff
@@ -18,11 +18,6 @@ data:
       resources:
       - group: ""
         resources: ["events"]
-    # Don't log oauth tokens as metadata.name is the secret
-    - level: None
-      resources:
-      - group: "oauth.openshift.io"
-        resources: ["oauthaccesstokens", "oauthauthorizetokens"]
```

Comment:
Can we remove the groups and go to userGroups key without issue?

Reply:
This is an array of specific configurations, I don't think there should be a problem removing an element, if that's what you're asking.